.github/workflows/upload_release_assets.yml

```diff
@@ -191,6 +191,15 @@ jobs:
         asset_path: ./assets/flow-aggregator.yml
         asset_name: flow-aggregator.yml
         asset_content_type: application/octet-stream
+    - name: Upload flow-visibility.yaml
```

Comment:
Do we need rename .yaml to .yml to keep consistence with other yaml files?

Reply:
renamed, thanks